Source/USemLog/Private/Control/SLControlManager.cpp

```diff
@@ -65,19 +65,11 @@ void ASLControlManager::ApplyForceTo(const FString& Id, FVector Force)
 
 	AActor* ActorToApply = Individual->GetParentActor();
 	UStaticMeshComponent* StaticMesh = Cast<UStaticMeshComponent>(ActorToApply->GetComponentByClass(UStaticMeshComponent::StaticClass()));
-	StaticMesh->AddForce(Force * 10000 * StaticMesh->GetMass());
+	StaticMesh->AddForce(Force * 100 * StaticMesh->GetMass());
```

Comment:
you can just use AddForce with the flag acceleration set to true, then it ignores the mass, if gives you more generality for any individual (heavier or lighter)